session.go

```diff
@@ -128,7 +128,7 @@ func NewSession(cfg ClusterConfig) (*Session, error) {
 	s.nodeEvents = newEventDebouncer("NodeEvents", s.handleNodeEvent)
 	s.schemaEvents = newEventDebouncer("SchemaEvents", s.handleSchemaEvent)
 
-	s.routingKeyInfoCache.lru = lru.New(cfg.MaxRoutingKeyInfo)
+	s.routingKeyInfoCache.lru = lru.New(cfg.MaxRoutingKeyInfo, time.Second)
```

Comment:
Perhaps it's safer to disable the LRU window update optimization for routing key info cache?

Reply:
I don't know. What would the possible consequences be?

Reply:
Disabling the LRU optimization for routing key info just means we revert to the old behavior, no? It's safer because we don't have to think about it. :)